MAUI/Maps/migration.md

```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
+    MapShapeSublayer sublayer = new MapShapeSublayer();
+    sublayer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/africa.json"));
+    sublayer.DataSource = viewModel.Data;
+    sublayer.PrimaryValuePath = "State";
+    sublayer.ShapeDataField = "name";
+    sublayer.ShapeColorValuePath = "Count";
+
+    EqualColorMapping colorMapping = new EqualColorMapping();
+    colorMapping.Color = Colors.Blue;
+    colorMapping.Value = "100";
+    colorMapping.Text = "100";
+
+    RangeColorMapping colorMapping1 = new RangeColorMapping();
+    colorMapping1.Color = Colors.Green;
+    colorMapping1.From = 0;
+    colorMapping1.To = 99;
+    colorMapping1.Text = "0-99";
+
+    sublayer.ColorMappings.Add(colorMapping);
+    sublayer.ColorMappings.Add(colorMapping1);
+
+    layer.Sublayers.Add(sublayer);
+    maps.Layer = layer;
+    this.Content = maps;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+## Vector layer
+
+### Polygon Layer
+
+<table>
+<tr>
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+<tr>
+   <td>
+      {{'[ShapeType](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeFileLayer.html#Syncfusion_SfMaps_XForms_ShapeFileLayer_ShapeTypeProperty)'| markdownify }}
+   </td>
+   <td>
+      {{'MapPolygon'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[Points](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeFileLayer.html#Syncfusion_SfMaps_XForms_ShapeFileLayer_PointsProperty)'| markdownify }}
+   </td>
+   <td>
+      {{'Points'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeFill](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeFill)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'Fill'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeStroke](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeStroke)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'Stroke'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeStrokeThickness](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeStrokeThickness)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'StrokeThickness'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```

Comment:
The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

Reply:
Modified